src/graphics/program-lib/programs/basic.js

```diff
@@ -9,6 +9,9 @@ import {
 
 import { begin, end, fogCode, precisionCode, skinCode } from './common.js';
 
+// Spector integration
+const shaderNameCode = '#define SHADER_NAME BasicMaterial\n';
```

Comment:
shouldn't this be MATERIAL_TYPE?

Reply:
Nope, Spector JS requires this name to capture / display it. I've used it in few other shaders already.